build/import/Versions.props

```diff
@@ -18,11 +18,8 @@
     <RoslynToolsRepoToolsetVersion>1.0.0-beta-62705-01</RoslynToolsRepoToolsetVersion>
     <RoslynToolsModifyVsixManifestVersion>1.0.0-beta-62327-02</RoslynToolsModifyVsixManifestVersion>
     <RoslynToolsSignToolVersion>1.1.0-beta3.21260.1</RoslynToolsSignToolVersion>
-    <RoslynDependenciesProjectSystemOptimizationDataVersion>2.6.0-beta1-62205-02</RoslynDependenciesProjectSystemOptimizationDataVersion>
-    <!--TODO: Merged PIAs - evaluate the need of this upgrade https://github.com/dotnet/project-system/issues/7191-->
-    <MicrosoftDotNetIBCMergeVersion>5.0.7-beta.20159.1</MicrosoftDotNetIBCMergeVersion>
```

Comment:
These directly related with the optimization that was done for Editors/AppDesigner assemblies. The use of optimization was disabled sometime in 2021 because of changes in package feeds. Daniel had attempted to work around this but it seems there were still issues with the workaround as the `/no-ibc` flag was still set on the build.cmd call for CI. This means that as of sometime in 2021, the Editors/AppDesigner DLLs are no longer optimized. Additional information I learned is in the comments on https://github.com/dotnet/project-system/issues/7191. Basically, the optimization data for these DLLs is over 4 years old and we'd want to use OptProf to do optimization at this point anyway. Thus, I removed the custom use of IBCMerge since it wasn't even being ran anymore anyway. A separate issue will be created for doing OptProf for Editors/AppDesigner as there are a few different tasks that need done for that to be implemented.